Python/PRP/Message/pyLinkToAgeMsg.cpp

```diff
@@ -46,8 +44,31 @@ PY_GETSET_SETTER_DECL(LinkToAgeMsg, ageLink)
 
 PY_PROPERTY_GETSET_DECL(LinkToAgeMsg, ageLink)
 
+PY_GETSET_GETTER_DECL(LinkToAgeMsg, linkEffects)
+{
+    // This cannot be a subclass, since it's an inline member
+    return pyAgeLinkEffects_FromAgeLinkEffects(&self->fThis->getLinkEffects());
+}
+
+PY_GETSET_SETTER_DECL(LinkToAgeMsg, linkEffects)
+{
+    PY_PROPERTY_CHECK_NULL(linkEffects)
+
+    plAgeLinkEffects effects = self->fThis->getLinkEffects();
```

Comment:
This needs to be a reference in order for the assignment below to work correctly...